deploy/olm-catalog/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,767 @@
+            "config": {
+              "authUniqueHosts": "internal-ip1 internal-ip2 mycluster.icp",
+              "clusterCADomain": "mycluster.icp",
+              "clusterExternalAddress": "10.0.0.1",
+              "clusterInternalAddress": "10.0.0.1",
+              "clusterName": "mycluster",
+              "defaultAdminPassword": "password",
+              "defaultAdminUser": "cpadmin",
+              "scimAdminPassword": "scimpassword",
+              "scimAdminUser": "scimadmin",
+              "enableImpersonation": false,
+              "fipsEnabled": true,
+              "ibmCloudSaas": false,
+              "icpPort": 8443,
+              "installType": "fresh",
+              "isOpenshiftEnv": true,
+              "nonceEnabled": true,
+              "xframeDomain": "",
+              "preferredLogin": "",
+              "bootstrapUserId": "kubeadmin",
+              "providerIssuerURL": "",
+              "claimsSupported": "name,family_name,display_name,given_name,preferred_username",
+              "claimsMap": "name=\"givenName\" family_name=\"givenName\" given_name=\"givenName\" preferred_username=\"displayName\" display_name=\"displayName\"",
+              "scopeClaim": "profile=\"name,family_name,display_name,given_name,preferred_username\"",
+              "oidcIssuerURL": "https://127.0.0.1:443/idauth/oidc/endpoint/OP",
+              "openshiftPort": 443,
+              "roksEnabled": true,
+              "roksURL": "https://roks.domain.name:443",
+              "roksUserPrefix": "changeme",
+              "saasClientRedirectUrl": "",
+              "wlpClientID": "4444be3a738841016ab76d71b650e836",
+              "wlpClientRegistrationSecret": "f1362ca4d20b8389af2d1ea68042c9af",
+              "wlpClientSecret": "aa73bf39752053bf723d1143fb4cf8a2"
+            },
+            "identityManager": {
+              "imageName": "icp-identity-manager",
+              "imageRegistry": "quay.io/opencloudio",
+              "imageTag": "4.1.0",
+              "masterNodesList": "10.0.0.1",
+              "resources": {
+                "limits": {
+                  "cpu": "1000m",
+                  "memory": "1Gi"
+                },
+                "requests": {
+                  "cpu": "50m",
+                  "memory": "150Mi"
+                }
+              }
+            },
+            "identityProvider": {
+              "imageName": "icp-identity-provider",
+              "imageRegistry": "quay.io/opencloudio",
+              "imageTag": "4.1.0",
+              "resources": {
+                "limits": {
+                  "cpu": "1000m",
+                  "memory": "1Gi"
+                },
+                "requests": {
+                  "cpu": "50m",
+                  "memory": "150Mi"
+                }
+              }
+            },
+            "initMongodb": {
+              "imageName": "icp-platform-auth",
+              "imageRegistry": "quay.io/opencloudio",
+              "imageTag": "3.10.0",
```

Comment:
Should be 4.1.0 ?

Reply:
@ujjwalchk-it , will take care in another PR. this is just alm-examples content. so no issues.